sys/net/network_layer/fib/fib.c

```diff
@@ -294,16 +299,19 @@ static int fib_signal_rrp(uint8_t *dst, size_t dst_size, uint32_t dst_flags)
     msg.content.ptr = (void *)&content;
 
     for (size_t i = 0; i < FIB_MAX_RRP; ++i) {
-        if (notify_rrp[i] != KERNEL_PID_UNDEF) {
+        if ((notify_rrp[i] != KERNEL_PID_UNDEF) && (prefix_rrp[i] != NULL)) {
```

Comment:
Does this check really need to happen if you're forcing protocols to supply their prefix anyway? If I understand it correctly, the purpose of this line is to weed out the “empty” entries, so just checking for ``KERNEL_PID_UNDEF `` should suffice, shouldn't it?